model/trigger/version.go

```diff
@@ -0,0 +1,130 @@
+package trigger
+
+import (
+	"fmt"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/event"
+	"github.com/evergreen-ci/evergreen/model/version"
+	restModel "github.com/evergreen-ci/evergreen/rest/model"
```

Comment:
why is there a dependency on the rest model in the db model package? this seems likely to cause strife in the future.

Reply:
This is taken from the previous notifications. We should push this as is, and discuss refactoring at some other point.

Reply:
I added a note to https://jira.mongodb.org/browse/EVG-3374.